server/controllers/payroll/staffingIndices/gradeIndice.js

```diff
@@ -16,6 +25,33 @@ function list(req, res, next) {
     }).catch(next);
 }
 
+function listRubrics(req, res, next) {
```

Comment:
This function name is confusing, since it doesn't list all rubrics.  Maybe you can rename it `listRubricsLinkedToGrade`?

Also, why can't this just be a query string parameter to the `list()` function?  Why do you need to make a whole other route?

I would assume that `GET /staffing_grade_indices/?is_linked_to_grade=1` should work, right?

Reply:
Good catch, i have delete it and use Rubrics.read(null, { is_linked_to_grade : 1 }) and it works well